.github/workflows/docs.yml

```
@@ -43,7 +43,7 @@ jobs:
         run: sudo apt-get install dia mscgen inkscape texlive-latex-recommended texlive-latex-extra
 
       - name: 🐍 Install Conda environment
-        uses: mamba-org/provision-with-micromamba@v15
+        uses: mamba-org/provision-with-micromamba@v16
```

```suggestion
        uses: mamba-org/setup-micromamba@v1
```
This should do the migration.

---

@pavelzw Thx, let have a try :)